go.mod

```diff
@@ -1,7 +1,10 @@
 module github.com/nats-io/nats-kafka
 
+go 1.13
```

Comment:
This all gets pulled in from running tests but we do not need them so just revert this file unless you are really adding new stuff.